openfe/utils/remove_oechem.py

```diff
@@ -1,28 +1,23 @@
 # This code is part of OpenFE and is licensed under the MIT license.
 # For details, see https://github.com/OpenFreeEnergy/openfe
-from openff.toolkit import GLOBAL_TOOLKIT_REGISTRY, OpenEyeToolkitWrapper
-from openff.toolkit.utils.toolkit_registry import ToolkitUnavailableException
+from openff.toolkit.utils.toolkit_registry import (
+    ToolkitRegistry,
+    toolkit_registry_manager,
+)
+from openff.toolkit.utils.toolkits import (
+    AmberToolsToolkitWrapper,
+    BuiltInToolkitWrapper,
```

Comment:
Nope

Reply:
😅 that's a relief

Reply:
With the verbosity afforded to me by a cup of coffee, the "functionality" it adds is zeroing out partial charges or assigning partial charges by the formal charge. There's good reason for at least one of these existing, but to my recollection only in some niche use cases that are far away from current mainline OpenFF force fields and/or some internal tests.

It's really only included here because it in `GLOBAL_TOOLKIT_REGISTRY` and removing is, strictly speaking, a behavior change.

https://github.com/openforcefield/openff-toolkit/blob/0.15.2/openff/toolkit/utils/builtin_wrapper.py#L83-L86